pkgs/applications/audio/spotifywm/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, spotify, xorg }:
+stdenv.mkDerivation rec {
+  name = "spotifywm-unstable-${version}";
+  version = "2016-11-28";
+
+  src = fetchFromGitHub {
+    owner  = "dasJ";
+    repo   = "spotifywm";
+    rev    = "91dd5532ffb7a398d775abe94fe7781904ab406f";
+    sha256 = "01z088i83410bpx1vbp7c6cq01r431v55l7340x3izp53lnpp379";
+  };
+
+  buildInputs = [ xorg.libX11 ];
+
+  propagatedBuildInputs = [ spotify ];
+
+  installPhase = ''
+    echo "#!/bin/sh" > spotifywm
```

Comment:
to ensure absolute purity you might want to use `#! ${stdenv.shell}` rather than relying on `/bin/sh`, right?

Reply:
Thanks, I just made the change.